<ui version="4.0" >
 <class>ArchiveDlg</class>
 <widget class="QDialog" name="ArchiveDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Conversations</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCalendarWidget" name="calendar" >
          <property name="verticalHeaderFormat" >
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
          <property name="dateEditEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lw_conversations" >
          <property name="contextMenuPolicy" >
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Logged messages</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>2</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="tw_log" >
          <property name="layoutDirection" >
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="indentation" >
           <number>4</number>
          </property>
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <property name="allColumnsShowFocus" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
          <property name="columnCount" >
           <number>3</number>
          </property>
          <column>
           <property name="text" >
            <string>1</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>2</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>3</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QToolButton" name="tb_prevPage" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="icon" >
             <iconset resource="../../iconsets.qrc" >:/iconsets/system/default/arrow_left.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_nextPage" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="icon" >
             <iconset resource="../../iconsets.qrc" >:/iconsets/system/default/arrow_right.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="value" >
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_close" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="icon" >
        <iconset resource="../../iconsets.qrc" >:/iconsets/system/default/close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../psi.qrc" />
  <include location="../../iconsets.qrc" />
 </resources>
 <connections/>
</ui>
